### PR TITLE
Implement Series.copy

### PR DESCRIPTION
**Pull request by itholic**
Implement Series.copy and add Examples to Frame.copy also

```python
>>> s = ks.Series([1, 2], index=["a", "b"])
>>> s
a    1
b    2
Name: 0, dtype: int64
>>> s_copy = s.copy()
>>> s_copy
a    1
b    2
Name: 0, dtype: int64
```

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -3313,13 +3313,29 @@ def to_records(self, index=True, convert_datetime64=None,
         return validate_arguments_and_invoke_function(
             kdf._to_internal_pandas(), self.to_records, pd.DataFrame.to_records, args)
 
+    # TODO: add 'deep' parameter
```
Comment (ueshin) [inline databricks/koalas/frame.py]: We will not support shallow copy. We can remove this.

File: databricks/koalas/series.py [hunk 2]
```diff
@@ -3108,6 +3108,30 @@ def idxmin(self, skipna=True):
         else:
             return tuple(values)
 
+    # TODO: add 'deep' parameter
```
Comment (ueshin) [inline databricks/koalas/series.py]: ditto.

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -3313,13 +3313,29 @@ def to_records(self, index=True, convert_datetime64=None,
...
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'x': [1, 2], 'y': [3, 4], 'z': [5, 6], 'w': [7, 8]},
+        ...                   columns=['x', 'y', 'z', 'w'])
+        >>> df
+           x  y  z  w
+        0  1  3  5  7
+        1  2  4  6  8
+        >>>
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: unneeded line?

Comment (itholic) [reply databricks/koalas/frame.py]: @ueshin , Thanks for comment :) Fixed them.

**Comment by ueshin** [comment]
Thanks! merging.

**Comment by itholic** [comment]
@ueshin Thanks :)

**Pull request merged**